### ISSUE TITLE
123

### ISSUE THREAD

**Issue by hvluu-8** [vllm-project/vllm#18295]
> are you refer to pip libs? here are them
> === System Information ===
> Platform: Linux-5.15.0-139-generic-x86_64-with-glibc2.35
> Python Version: 3.10.12 (main, Feb  4 2025, 14:57:36) [GCC 11.4.0]
> Python Executable: /data/dwenlong/SGPT/bin/python
> 
> === Installed pip Packages ===
> Package                                  Version       Editable project location
> ---------------------------------------- ------------- ---------------------------------
> absl-py                                  2.1.0
> accelerate                               1.7.0
> aiohappyeyeballs                         2.6.1
> aiohttp                                  3.11.18
> aiosignal                                1.3.2
> airportsdata                             20250224
> annotated-types                          0.7.0
> antlr4-python3-runtime                   4.7.2
> anyio                                    4.9.0
> astor                                    0.8.1
> async-timeout                            5.0.1
> attrs                                    25.1.0
> black                                    22.8.0
> blake3                                   1.0.4
> cachetools                               5.5.2
> certifi                                  2025.1.31
> cffi                                     1.16.0
> charset-normalizer                       3.4.1
> click                                    8.1.8
> cloudpickle                              3.1.1
> cmake                                    4.0.2
> compressed-tensors                       0.9.2
> contextlib2                              21.6.0
> contourpy                                1.2.1
> coverage                                 7.6.12
> cupy-cuda12x                             12.1.0
> cycler                                   0.12.1
> datasets                                 3.6.0
> deepspeed                                0.16.0
> Deprecated                               1.2.18
> depyf                                    0.18.0
> dill                                     0.3.8
> diskcache                                5.6.3
> distro                                   1.9.0
> dnspython                                2.7.0
> einops                                   0.8.0
> email_validator                          2.2.0
> et-xmlfile                               1.1.0
> exceptiongroup                           1.2.2
> fastapi                                  0.115.12
> fastapi-cli                              0.0.7
> fastrlock                                0.8.3
> filelock                                 3.18.0
> flash_attn                               2.7.4.post1
> fonttools                                4.53.1
> frozenlist                               1.5.0
> fsspec                                   2024.6.1
> gguf                                     0.10.0
> googleapis-common-protos                 1.70.0
> grpcio                                   1.70.0
> h11                                      0.16.0
> hjson                                    3.1.0
> httpcore                                 1.0.9
> httptools                                0.6.4
> httpx                                    0.28.1
> huggingface-hub                          0.31.2
> idna                                     3.10
> importlib_metadata                       8.0.0
> iniconfig                                2.0.0
> interegular                              0.3.3
> Jinja2                                   3.1.6
> jiter                                    0.9.0
> joblib                                   1.4.2
> jsonschema                               4.23.0
> jsonschema-specifications                2024.10.1
> kiwisolver                               1.4.5
> lark                                     1.2.2
> latex2sympy2                             1.9.1
> llguidance                               0.7.20
> llvmlite                                 0.43.0
> lm-format-enforcer                       0.10.11
> loguru                                   0.7.3
> Markdown                                 3.7
> markdown-it-py                           3.0.0
> MarkupSafe                               2.1.5
> matplotlib                               3.9.1
> mdurl                                    0.1.2
> mistral_common                           1.5.4
> ml-collections                           0.1.1
> mpmath                                   1.3.0
> msgpack                                  1.1.0
> msgspec                                  0.19.0
> multidict                                6.4.3
> multiprocess                             0.70.16
> munch                                    4.0.0
> mypy-extensions                          1.0.0
> nest-asyncio                             1.6.0
> networkx                                 3.3
> ninja                                    1.11.1.3
> numba                                    0.60.0
> numpy                                    1.26.4
> nvidia-cublas-cu12                       12.4.5.8
> nvidia-cuda-cupti-cu12                   12.4.127
> nvidia-cuda-nvrtc-cu12                   12.4.127
> nvidia-cuda-runtime-cu12                 12.4.127
> nvidia-cudnn-cu12                        9.1.0.70
> nvidia-cufft-cu12                        11.2.1.3
> nvidia-curand-cu12                       10.3.5.147
> nvidia-cusolver-cu12                     11.6.1.9
> nvidia-cusparse-cu12                     12.3.1.170
> nvidia-cusparselt-cu12                   0.6.2
> nvidia-ml-py                             12.570.86
> nvidia-nccl-cu12                         2.21.5
> nvidia-nvjitlink-cu12                    12.4.127
> nvidia-nvtx-cu12                         12.4.127
> omegaconf                                2.3.0
> openai                                   1.79.0
> opencv-python-headless                   4.11.0.86
> openpyxl                                 3.1.5
> opentelemetry-api                        1.26.0
> opentelemetry-exporter-otlp              1.26.0
> opentelemetry-exporter-otlp-proto-common 1.26.0
> opentelemetry-exporter-otlp-proto-grpc   1.26.0
> opentelemetry-exporter-otlp-proto-http   1.26.0
> opentelemetry-proto                      1.26.0
> opentelemetry-sdk                        1.26.0
> opentelemetry-semantic-conventions       0.47b0
> opentelemetry-semantic-conventions-ai    0.4.9
> orz                                      0.1.0         /data/dwenlong/Open-Reasoner-Zero
> outlines                                 0.1.11
> outlines_core                            0.1.26
> packaging                                24.1
> pandas                                   2.2.2
> partial-json-parser                      0.2.1.1.post5
> pathspec                                 0.12.1
> Pebble                                   5.1.1
> pillow                                   10.4.0
> pip                                      22.0.2
> platformdirs                             4.3.6
> pluggy                                   1.5.0
> pretrainedmodels                         0.7.4
> prometheus_client                        0.22.0
> prometheus-fastapi-instrumentator        7.1.0
> propcache                                0.3.1
> protobuf                                 3.20.3
> psutil                                   7.0.0
> py-cpuinfo                               9.0.0
> pyarrow                                  20.0.0
> pycountry                                24.6.1
> pycparser                                2.22
> pydantic                                 2.10.6
> pydantic_core                            2.27.2
> Pygments                                 2.19.1
> pynvml                                   11.5.0
> pyparsing                                3.1.2
> pytest                                   8.3.4
> pytest-cov                               6.0.0
> python-dateutil                          2.9.0.post0
> python-dotenv                            1.1.0
> python-json-logger                       3.3.0
> python-multipart                         0.0.20
> python_papi                              6.0.0.1
> pytz                                     2024.1
> PyYAML                                   6.0.1
> pyzmq                                    26.4.0
> ray                                      2.46.0
> referencing                              0.36.2
> regex                                    2024.11.6
> requests                                 2.32.3
> rich                                     14.0.0
> rich-toolkit                             0.14.6
> rpds-py                                  0.23.1
> safetensors                              0.5.2
> scikit-learn                             1.5.1
> scipy                                    1.14.0
> sentencepiece                            0.2.0
> setuptools                               59.6.0
> shellingham                              1.5.4
> six                                      1.16.0
> sniffio                                  1.3.1
> starlette                                0.46.2
> sympy                                    1.13.1
> tabulate                                 0.9.0
> tensorboard                              2.19.0
> tensorboard-data-server                  0.7.2
> tensorboardX                             2.6.2.2
> threadpoolctl                            3.5.0
> tiktoken                                 0.9.0
> timeout-decorator                        0.5.0
> tokenizers                               0.21.1
> tomli                                    2.2.1
> torch                                    2.6.0
> torchaudio                               2.6.0
> torchvision                              0.21.0
> tqdm                                     4.66.4
> transformers                             4.51.3
> triton                                   3.2.0
> typer                                    0.15.4
> typing_extensions                        4.12.2
> tzdata                                   2024.1
> urllib3                                  2.3.0
> uvicorn                                  0.34.2
> uvloop                                   0.21.0
> vllm                                     0.8.2
> vllm-nccl-cu12                           2.18.1.0.4.0
> watchfiles                               1.0.5
> websockets                               15.0.1
> Werkzeug                                 3.1.3
> word2number                              1.1
> wrapt                                    1.17.2
> xformers                                 0.0.29.post2
> xgrammar                                 0.1.16
> xxhash                                   3.5.0
> yarl                                     1.20.0
> zipp                                     3.21.0
>  

 _Originally posted by @vengdeng in [#15773](https://github.com/vllm-project/vllm/issues/15773#issuecomment-2888015737)_

**Comment by DarkLight1337** [comment]
I think you opened this by mistake
